controllers/cards.js

```diff
@@ -0,0 +1,27 @@
+const Card = require('../models/card');
+
+const showAllCards = (req, res) => {
+  Card.find({})
+    .populate(['owner', 'likes'])
+    .then((cards) => res.send({ data: cards }))
+    .catch(() => res.status(500).send({ message: 'Произошла ошибка' }));
+};
+
+const postCard = (req, res) => {
+  const { name, link } = req.body;
+  const { _id } = req.user;
+
+  Card.create({
+    name, link, owner: _id, likes: _id,
+  })
+    .then((ad) => res.send({ data: ad }))
+    .catch((err) => res.status(500).send({ message: err }));
+};
+
+const deleteCard = (req, res) => {
+  Card.findByIdAndRemove(req.params._id)
+    .then((card) => (!card ? res.status(404).send({ message: 'Произошла ошибка' }) : res.send({ data: card })))
```

Comment:
Можно лучше: отправлять корректный ответ.
Когда запись не найдена лучше сообщить, что запись не найдена. Формулировка _Произошла ошибка_ говорит клиенту немного о другом.
```suggestion
    .then((card) => (!card ? res.status(404).send({ message: 'Нет карточки с таким id' }) : res.send({ data: card })))
```